reflex/state.py

```diff
@@ -1479,6 +1511,8 @@ async def __aenter__(self) -> StateProxy:
         """
         self._self_actx = self._self_app.modify_state(
             self.__wrapped__.router.session.client_token
+            + "_"
```

Comment:
same comment about pulling this out

Reply:
adding a utility for this in the followup PR